pygitguardian/iac_models.py

```diff
@@ -16,6 +17,11 @@ class IaCVulnerability(Base, FromDictMixin):
     documentation_url: str
     component: str = ""
     severity: str = ""
+    url: Optional[str] = None
+    status: Optional[str] = None
+    ignored_until: Optional[datetime] = None
+    ignore_reason: Optional[str] = None
+    ignore_comment: Optional[str] = None
```

Comment:
The new optional string fields are `Optional[str] = None`, whereas the existing optional string fields (`component`, `severity`) are `str = ""`. If there is no reason to differentiate `""` and `None`, I would prefer if we used `str = ""` everywhere.

Reply:
I think it makes sense to differentiate them, as `component` and `severity` are set for all IaCVulnerability, and these new fields are not.
I've made the suggested change, but maybe we should remove the default values for `component` and `severity` instead? WDYT?

Reply:
(Sorry for the late reply)

Indeed, if `component` and `severity` are always set, then it's better to make them mandatory, without a default value. And we can keep the `Optional[str]` to be consistent with the `Optional[datetime]`.